01-upload-a-file/python/es1_upload.py

```diff
@@ -0,0 +1,23 @@
+# http://elementalselenium.com/tips/1-upload-a-file
+import os
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys # for send_keys
```

Comment:
Since this example doesn't use the Keys module, it's a good idea to leave it out until it's used in an example.